package-lock.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Wrong package lock file.
It needs to be the one in auth-web/.

Reply:
(delete this one)

Reply:
Removed